server.go

```diff
@@ -954,6 +959,9 @@ func (s *server) Start() error {
 	if err := s.utxoNursery.Start(); err != nil {
 		return err
 	}
+	if err := s.channelNotifier.Start(); err != nil {
```

Comment:
we may want to move this service up in the start order, to ensure that we don't miss any notifications. perhaps after the chain notifier?

Reply:
good point, fixed